crates/rpc/rpc/src/trace.rs

```diff
@@ -408,6 +411,43 @@ where
             )
             .await?
             .map(|traces| traces.into_iter().flatten().collect());
+
+        // Add block reward traces
+        // TODO: We only really need the header and ommers here to determine the reward
+        if let (Some(block), Some(traces)) =
+            (self.inner.eth_api.block_by_id(block_id).await?, traces.as_mut())
+        {
+            if let Some(header_td) = self.provider().header_td(&block.header.hash)? {
+                if let Some(base_block_reward) = base_block_reward(
```

Comment:
if this is none we are past the merge and there are no block rewards and no ommers